addon/components/bulma-radio.js

```diff
@@ -21,7 +20,8 @@ export default BulmaInput.extend({
   init() {
     this._super(...arguments);
 
-    // Remove the inherited input class name (as it breaks the styling)
-    emberArray(get(this, 'classNames')).removeObject('input');
+    // classNames reset
+    // Remove the inherited `input` class name (as it breaks the styling)
+    set(this, 'classNames', ['radio']);
```

Comment:
@alexdiliberto This wouldn't override any classnames when`attrs` are received right?

```hbs
{{bulma-radio classNames="custom-radio-class"}}
```